docs/modules/configuration/pages/dynamic-config-persistence.adoc

```diff
@@ -0,0 +1,173 @@
+= Dynamic Configuration Persistence
+:description: You can configure members to keep dynamic configuration changes in memory or persist them in a YAML or XML configuration file.
+
+{description}
+
+NOTE: Please make sure to give write permissions to the configuration file.
+
+== Enabling Dynamic Configuration Persistence
+
+To enable dynamic configuration persistence adding the following configuration is enough:
+
+[tabs]
+====
+XML::
++
+--
+```xml
+<hazelcast>
+  <dynamic-configuration>
+    <persistence-enabled>true</persistence-enabled>
+</hazelcast>
+```
+--
+YAML::
++
+--
+```yml
+hazelcast:
+  dynamic-configuration:
+    persistence-enabled: true
+```
+--
+====
+
+By default, the xref:configuring-declaratively.adoc[declarative configuration file] used to persist dynamic changes. This also means that you need to use hazelcast with a configuration file.
+
```

Comment:
> Do you use <<snippets, configuration snippets>>?

What this item mean? Why it is this asked?

Reply:
because if a user uses snippets, they may want to add a change to a snippet instead of the root config file, in which case the user can configure `peristence-file`